css/footer.css

```diff
@@ -90,9 +98,7 @@ h4.info-block {
     font-size: 18px;
     letter-spacing: 1px; 
 }
-.row > .col-4{
```

Comment:
This change overrides the original layout.css file. It will bring changes to all our layout. Are you sure its correct?